types/handlebars/handlebars-tests.ts

```diff
@@ -1,6 +1,6 @@
 
 //import Handlebars = require('handlebars');
-import * as Handlerbars from 'handlebars';
+import * as Handlebars from 'handlebars';
```

Comment:
This typo is good evidence that handlebars should switch to the new global syntax:

``ts
export = Handlebars;
export as namespace Handlebars;
```

Reply:
Thanks for the feedback! In parallel, I've been working to have Handlebars provide its own typings as part of the package (https://github.com/wycats/handlebars.js/pull/1487), so I'll look into making this change over there (likely in a follow-up PR).